packages/core/useMouse/demo.vue

```diff
@@ -9,11 +9,11 @@ const parentEl = useParentElement()
 const mouseDefault = reactive(useMouse())
 const textDefault = stringify(mouseDefault)
 
-const extractor: UseMouseEventExtractor = (event) => {
-  if (typeof Touch !== 'undefined' && event instanceof Touch)
+const extractor: UseMouseEventExtractor = (e) => {
+  if (typeof Touch !== 'undefined' && e instanceof Touch)
     return null
-  else
-    return [event.offsetX, event.offsetY]
+  const event = e as MouseEvent
+  return [event.offsetX, event.offsetY]
```

Comment:
@antfu this will now conflict with https://github.com/vueuse/vueuse/pull/4457. #4457 should solve this type issue without a cast